public/challenge-contributions/1/asavov.html

```diff
@@ -4,8 +4,46 @@
   <meta charset="UTF-8">
```

Comment:
Please validate your HTML, there is a minor thing you should update. https://validator.w3.org/#validate_by_input

Reply:
sorry but I don't understand what you actually want here